composer.json

```diff
@@ -23,7 +23,7 @@
         "knplabs/knp-snappy-bundle": "^1.7",
         "myclabs/php-enum": "^1.7",
         "sylius/resource-bundle": "^1.6",
-        "sylius/sylius": "^1.9",
+        "sylius/sylius": "^1.9 || ~1.10.0@beta",
```

Comment:
Will this allow to install the version 1.10.0?

Reply:
I was suggesting myself by utilising -> https://github.com/Sylius/PluginSkeleton/blob/1.10/composer.json#L12